includes/models/model.llms.quiz.php

```diff
@@ -108,6 +109,27 @@ public function get_questions( $return = 'questions' ) {
 		return $this->questions()->get_questions( $return );
 	}
 
+	/**
+	 * Get questions count.
+	 *
+	 * @since [version]
+	 *
+	 * @return int Question Count.
+	 */
+	public function get_questions_count() {
+
+		if ( $this->get( 'question_bank' ) ) {
+
+			$questions_limit = $this->get( 'number_of_questions' );
+
+			if ( $questions_limit ) {
+				return $questions_limit;
+			}
+		}
+
+		return count( $this->get_questions( 'ids' ) );
+	}
```

Comment:
Most if the logic should be moved into the add-on.
Here you can create a filter to be leveraged from the add-on.

Reply:
moved it. thanks